### DISCUSSION TITLE
When using lammps to run MD, an error is reported

### DISCUSSION THREAD

**Discussion by chb-deep** [deepmodeling/deepmd-kit#2645] [Q&A]
in.lammps:
# gas phase methane

units           metal
boundary        p p p
atom_style      atomic

neighbor        1.0 bin
neigh_modify    every 10 delay 0 check no

read_data	conf.lmp
mass 		1 1
mass		2 12

pair_style	deepmd CH4-compress.pb
pair_coeff	* *

velocity        all create 50.0 23456789
fix             1 all nvt temp 50.0 50.0 0.5
timestep        0.001

thermo_style    custom step pe ke etotal temp press vol
thermo          100
dump		1 all custom 100 ch4.dump id type x y z 

run             5000

err:
(deepmd1) [shuoxing@node01 02.lmp]$ ln -s ../slurmtijiao1/CH4-compress.pb
(deepmd1) [shuoxing@node01 02.lmp]$ lmp -i in.lammps 
LAMMPS (23 Jun 2022 - Update 1)
OMP_NUM_THREADS environment is not set. Defaulting to 1 thread. (src/comm.cpp:98)
  using 1 OpenMP thread(s) per MPI task
Reading data file ...
  triclinic box = (0 0 0) to (10.114259 10.263124 10.216793) with tilt (0.036749877 0.13833062 -0.056322169)
  1 by 1 by 1 MPI processor grid
  reading atoms ...
  5 atoms
  read_data CPU = 0.002 seconds
ERROR: Unrecognized pair style 'deepmd' (src/force.cpp:279)
Last command: pair_style	deepmd CH4-compress.pb

**Comment by njzjz** [comment]
Please provide detailed steps to reproduce how you installed it.
